snafu/benchmarks/dnsperf/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM registry.centos.org/centos:8
+
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+
+# snafu dependencies
+## download, compile python 3.7
+RUN dnf install -y \
+        bzip2-devel \
+        gcc \
+        libffi-devel \
+        make \
+        openssl-devel \
+        xz-devel \
+        zlib-devel && \
+    curl https://www.python.org/ftp/python/3.7.7/Python-3.7.7.tgz -o /opt/Python-3.7.7.tgz && \
+    tar xzf /opt/Python-3.7.7.tgz --directory /opt
+
+RUN dnf install -y \
+        git \
+        redis && \
+        curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz -C /usr/bin/ oc
+
+WORKDIR /opt/Python-3.7.7
+
+RUN ./configure --enable-optimizations && \
+    make altinstall && \
+    ln -s /usr/local/bin/python3.7 /usr/local/bin/python && \
+    python -m pip install -U pip && \
+    mkdir --parent /opt/snafu
+
+COPY . /opt/snafu
+
+RUN python -m pip install -e /opt/snafu
+
+# dnsperf dependencies
+RUN dnf install -y --nodocs epel-release && \
```

Comment:
Can we move this RUN command to be before the snafu installation? This way during local development/testing if we change something to snafu we don't have to also reinstall dnsperf: https://gabnotes.org/docker-images-layers-and-cache/. Maybe move all of these dnf steps into the first RUN command?